packages/TelegramClient-Core.package/TCCChatsHandler.class/instance/chatHistoryReceived..st

```diff
@@ -6,13 +6,4 @@ chatHistoryReceived: aChatHistory
 	(aChatHistory at: 'messages') ifNotEmpty: [
 		chatID := (aChatHistory at: 'messages') first at: 'chat_id'.
 		chat := self core chats getChat: chatID.
-
-		(aChatHistory at: 'messages') do: [:message | chat addOldestMessage: (TCCMessage newFromMessageEvent: message in: chat with: self core)].
-
-		chat waitingForUpdate: false.
-
-		chat stillRequestedMessages > 0
-			ifTrue: [self
-				getChatHistoryFrom: chatID
-				with: chat messages last id
-				limit: chat stillRequestedMessages]].
\ No newline at end of file
+		chat chatHistoryReceived: aChatHistory].
```

Comment:
`chatID` and `chat` are only used once now

Reply:
I think removing both would result in one very long and undreadable line. So i suggest leaving `chatID` in, do you agree with that?